### PR TITLE
Docstrings -> Google style for mkdocs render

### PR DESCRIPTION
**Pull request by CBroz1**
1. Change docstrings to google style
2. add typing information for rendering API tables

### CODE REVIEW

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @CBroz1.  Please also update the readme to simplify similar to the recent changes in https://github.com/datajoint/element-deeplabcut/pull/53.

**Comment by kabilar** [comment]
Please add a class docstring for the `Device` table in `pipeline.py`.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 
 ## 0.1.1 - Unreleased
```
Comment (kabilar) [inline CHANGELOG.md]:
```suggestion
## [0.1.1] - 2022-10-27
```

File: README.md [hunk 2]
```diff
@@ -1,72 +1,14 @@
...
+This repository demonstrates the functionality of the 
+DataJoint Element for markerless pose estimation with
+[DeepLabCut](https://www.deeplabcut.org/).  DataJoint Elements collectively standardize
+and automate data collection and analysis for neuroscience experiments.  Each Element is
+a modular pipeline for data storage and processing with corresponding database
+tables that can be combined with other Elements to assemble a fully functional pipeline.
+
+Installation and usage instructions, including those featured in this repository,
+can be found at the 
+[Element documentation](datajoint.com/docs/elements/element-deeplabcut).
```
Comment (kabilar) [inline README.md]:
```suggestion
Installation and usage instructions can be found at the 
[Element documentation](datajoint.com/docs/elements/element-deeplabcut).
```

File: README.md [hunk 3]
```diff
@@ -1,72 +1,14 @@
...
-    + Yatsenko D, Nguyen T, Shen S, Gunalan K, Turner CA, Guzman R, Sasaki M, Sitonic D, Reimer J, Walker EY, Tolias AS. DataJoint Elements: Data Workflows for Neurophysiology. bioRxiv. 2021 Jan 1. doi: https://doi.org/10.1101/2021.03.30.437358
-
-    + DataJoint Elements ([RRID:SCR_021894](https://scicrunch.org/resolver/SCR_021894)) - Element DeepLabCut (version `<Enter version number>`)
\ No newline at end of file
+This repository demonstrates the functionality of the 
+DataJoint Element for markerless pose estimation with
+[DeepLabCut](https://www.deeplabcut.org/).  DataJoint Elements collectively standardize
+and automate data collection and analysis for neuroscience experiments.  Each Element is
+a modular pipeline for data storage and processing with corresponding database
+tables that can be combined with other Elements to assemble a fully functional pipeline.
```
Comment (kabilar) [inline README.md:8]:
```suggestion
The DataJoint Workflow for DeepLabCut combines multiple DataJoint Elements for markerless pose estimation with
[DeepLabCut](https://www.deeplabcut.org/).  DataJoint Elements collectively standardize
and automate data collection and analysis for neuroscience experiments.  Each Element is
a modular pipeline for data storage and processing with corresponding database
tables that can be combined with other Elements to assemble a fully functional pipeline.
```

File: CHANGELOG.md (hunk 1 above)
Comment (kabilar) [inline CHANGELOG.md]: Please also add the link to the bottom of the file:
```
[0.1.1]: https://github.com/datajoint/workflow-deeplabcut/releases/tag/0.1.1
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @CBroz1.  I have reviewed a couple of files and have minor comments below.

@kushalbakshi Please review the docstrings and merge when you are ready.

**Review by kushalbakshi — APPROVED** [review]

**Pull request merged**